pci_dss_v321/pci_dss.sp

```diff
@@ -0,0 +1,26 @@
+locals {
+  pci_dss_v321_common_tags = merge(local.azure_compliance_common_tags, {
+    pci_dss_v321 = "true"
+    type         = "Benchmark"
+  })
+}
+
+benchmark "pci_dss_v321" {
+  title       = "PCI DSS 3.2.1"
+  description = "The PCI DSS applies to all entities that store, process, and/or transmit cardholder data. It covers technical and operational system components included in or connected to cardholder data. If you accept or process payment cards, PCI DSS applies to you."
+  documentation = file("./pci_dss_v321/docs/pci_dss_v321_overview.md")
```

Comment:
benchmark "pci_dss_v321" {
  title         = "PCI DSS 3.2.1"
  description   = "The PCI DSS applies to all entities that store, process, and/or transmit cardholder data. It covers technical and operational system components included in or connected to cardholder data. If you accept or process payment cards, PCI DSS applies to you."
  documentation = file("./pci_dss_v321/docs/pci_dss_v321_overview.md")
```suggestion
benchmark "pci_dss_v321" {
  title         = "PCI DSS 3.2.1"
  description   = "The PCI DSS applies to all entities that store, process, and/or transmit cardholder data. It covers technical and operational system components included in or connected to cardholder data. If you accept or process payment cards, PCI DSS applies to you."
  documentation = file("./pci_dss_v321/docs/pci_dss_v321_overview.md")
```